src/main/groovy/wooga/gradle/paket/unity/tasks/PaketUnityInstall.groovy

```diff
@@ -95,19 +96,40 @@ class PaketUnityInstall extends ConventionTask implements PaketUpmPackageSpec {
     @Input
     AssemblyDefinitionFileStrategy assemblyDefinitionFileStrategy
 
-    // We need this cache, since the mapping from nuget to upm package Id exists only in the package.json of a package from the paket packages cache
-    // since this can get deleted, we need to look inside the not-yet-deleted unity upm package and deduce the nuget & paket Id from there.
+    /**
+     * We need this cache, since the mapping from nuget to upm package Id exists only in the package.json of a package from the paket packages cache
+     * since this can get deleted, we need to look inside the not-yet-deleted unity upm package and deduce the nuget & paket Id from there.
+     */
     @Internal
-    NugetToUpmPackageIdCache nugetToUpmId
+    NugetToUpmPackageIdCache nugetToUpmCache
 
     public final static String assemblyDefinitionFileExtension = "asmdef"
 
     /**
-     * @return the installation output directory
+     * @return The installation output directory
      */
     @OutputDirectory
     File getOutputDirectory() {
-        new File(getReferencesFile().getParentFile(), "Assets/${getPaketOutputDirectoryName()}")
+        // TODO: Update. How the heck does this work nowadays when the Unity project directory is at the root??
```

Comment:
Remove this todo, or at least replace if by an comment answering it ;)

Reply:
It has been refactored!!